cmd/loadtest/loadtest.go

```diff
@@ -33,41 +32,96 @@ import (
 	"golang.org/x/time/rate"
 )
 
+//go:generate stringer -type=loadTestMode
+type (
+	loadTestMode int
+)
+
 const (
-	loadTestModeTransaction          = "t"
-	loadTestModeDeploy               = "d"
-	loadTestModeCall                 = "c"
-	loadTestModeFunction             = "f"
-	loadTestModeInc                  = "i"
-	loadTestModeRandom               = "r"
-	loadTestModeStore                = "s"
-	loadTestModeERC20                = "2"
-	loadTestModeERC721               = "7"
-	loadTestModePrecompiledContracts = "p"
-	loadTestModePrecompiledContract  = "a"
-	loadTestModeRecall               = "R"
+	loadTestModeTransaction loadTestMode = iota
+	loadTestModeDeploy
+	loadTestModeCall
+	loadTestModeFunction
+	loadTestModeInc
+	loadTestModeStore
+	loadTestModeERC20
+	loadTestModeERC721
+	loadTestModePrecompiledContracts
+	loadTestModePrecompiledContract
+	loadTestModeRecall
+
+	// All the modes AFTER random mode will not be used when mode random is selected
```

Comment:
should we communicate this behavior to user via help cmd?

Reply:
I think this is more of an implementation detail that ideally shouldn't concern a user. E.g. if you add a new mode like `loadTestUniswap` before `loadTestModeRandom` it will be eligible to be called during random mode testing. If you add it after `loadTestModeRandom` it will not be called ruing random mode. This is a quirk of using `iota` but in this case I'm using it to exclude `loadTestModeRPC` from random mode because it depends on only doing calls. 

Either way, i would like to improve the docs of the modes because there are a lot of features and capabilities that are hard to understand.